### DISCUSSION TITLE
Changing Password Timeout

### DISCUSSION THREAD

**Discussion by estreetworkshop** [kimai/kimai#4175] [Questions & Answers]
Hello,

I just upgraded to Kimai 2.0.  It is asking for my password to unlock my account throughout the day.  Is there a way to change the password timeout to something longer?  I do not see a setting for it in the admin page.

Thank you!

**Comment by kevinpapst** [comment]
It is your PHP session timeout setting that's causing it.

It is by default configured to 14 minutes, I'd recommend to raise it to a week or so ...

**Comment by estreetworkshop** [comment]
Thank you!

For anyone else that needs this.  I had to change following values in php.ini.  Set XXXX to the seconds of the timeout.

session.gc_maxlifetime = XXXX
session.cookie_lifetime = XXXX
